### ISSUE TITLE
Deleting a tag in the Android app does not delete it from the web or desktop version

### ISSUE THREAD

**Issue by rachelsquirrel** [Automattic/simplenote-android#279]
Steps:
1. Open the Android app
2. Tap to open the menu at the top left
3. Tap the Edit link next to Tags
4. Delete a tag

The tag is deleted from the list of tags in the app but is still available from other Simplenote apps like IOS, web, and desktop.

**Comment by melines** [comment]
Another report: ticket #1508255 zen. The user mentioned sync missing between windows and android, regardless of the device used to delete the tags.

**Comment by mrfoxtalbot** [comment]
Another possible report here: 2029492-zen

The app version is Simplenote Version 1.7.0 and OS is Andoid 7.0.

**Comment by fresatomica** [comment]
Another one reported here: #2033374-zen

They are using Simplenote 1.7.0 on Xiaomi MI A1 with Android 9

**Comment by theck13** [comment]
I reproduced that behavior using a Google Pixel 3 on Android 9.0 with Simplenote 1.7.0.  I confirmed the following:
- Tag created on Android that is deleted on Android or web is removed on Android and web.
- Tag created on web that is deleted on web is removed on Android and web.
- Tag created on web that is deleted on Android is removed on Android, but not web.
    - Logging out/in causes the tag to reappear in tag list on Android.
    - Tag is removed from notes on Android and web.
